src/components/DeleteOrg/DeleteOrg.tsx

```diff
@@ -29,24 +30,26 @@ function deleteOrg(): JSX.Element {
     },
   });
 
+  const history = useHistory();
+
   const deleteOrg = async (): Promise<void> => {
     if (data && data.isSampleOrganization) {
-      removeSampleOrganization()
-        .then(() => {
-          toast.success('Successfully deleted sample Organization');
-        })
-        .catch((error) => {
-          toast.error(error.message);
-        });
-      window.location.replace('/orglist');
+      try {
+        await removeSampleOrganization();
+        toast.success('Successfully deleted sample Organization');
+        history.push('/orglist');
+      } catch (error: any) {
+        toast.error(error.message);
+      }
```

Comment:
Please write the test for these lines.

Reply:
I tried to write the tests, but every time I got an unexpected error that I was not able to solve.
According to my knowledge, my basics are correct but then also tests are not passing.